libraries/AP_HAL_ChibiOS/hwdef/JHEMCU-GSF405A/hwdef.dat

```diff
@@ -0,0 +1,149 @@
+# hw definition file for processing by chibios_pins.py
+# JHEMCU GSF405A by V-22
+# With F405 MCU, MPU6000 IMU and MAX7456 series OSD
+# Based on Mamba F405 hwdef from jeanphilippehell
+# thanks to betaflight for pin information (https://github.com/betaflight/unified-targets/blob/master/configs/default/JHEF-JHEF405PRO.config)
```

Comment:
My guess is that you should set FRAME_TYPE to betaflight X - have you tried this on an actual quad? Do the motors come out in the right order?

Reply:
I was planning on putting this in a new build, but the parts I have planned do not have an ETA currently. I have another build I can swap this into for testing, and hope to do so this weekend.